tests/linalg_test.py

```diff
@@ -1318,7 +1318,10 @@ def testExpmGrad(self, n, dtype, rng_factory):
      rng = rng_factory(self.rng())
      jtu.skip_if_unsupported_type(dtype)
      a = rng((n, n), dtype)
-     jtu.check_grads(jsp.linalg.expm, (a,), modes=["fwd"], order=1)
+     def expm(x):
+       return jsp.linalg.expm(x, upper_triangular=False, max_squarings=16)
+     jtu.check_grads(expm, (a,), modes=["fwd", "rev"], order=1)
```

Comment:
You can remove this line. `order=2` means "all orders up to 2", so this is covered in the line below.

Reply:
Done.